<launch>
	<node pkg="copy_when_uncertain" type="environment_updater.py" name="environment_updater"/>
	<node pkg="copy_when_uncertain" type="evolve_server.py" name="evolve_server"/>
	<node pkg="copy_when_uncertain" type="record_generation_server.py" name="record_generation_server">
		<param name="filename_prefix" value="high_variance"/>
	</node>
</launch>         
